http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala

```diff
@@ -84,7 +84,8 @@ final case class ClusterRequest(labels: LabelMap = Map.empty,
                                 rstudioDockerImage: Option[String] = None,
                                 welderDockerImage: Option[String] = None,
                                 scopes: Set[String] = Set.empty,
-                                enableWelder: Option[Boolean] = None)
+                                enableWelder: Option[Boolean] = None,
+                                customClusterEnvironmentVariables: Map[String, String] = Map.empty[String, String])
```

Comment:
You probably also need to update the json serializer `ClusterRequestFormat`, also in this file. This field can be handled the same way as `labels`.

Reply:
Want me to make an `EnvironmentVariableMap` type for this too or just leave it as a `Map[String, String]`?

Reply:
Nah, leave it as `Map[String, String]`. I don't think the type alias adds any value.

Btw, you can probably just write `Map.empty` - it should infer the types.

Reply:
Scala magic!